.github/workflows/token-dashboard.yml

```diff
@@ -0,0 +1,87 @@
+name: Token Dashboard dApp
+
+#TODO: extend the conditions once workflow gets tested together with other workflows
+on:
+  push:
+    branches:
+      # TODO: Run on master after we're fully migrated from Circle CI
+      # - master
+      - "rfc-18/**"
+    paths:
+      - "solidity/dashboard/**"
+      - ".github/workflows/token-dashboard.yml"
+  pull_request:
+    branches:
+      # TODO: Run on all branches after we're fully migrated from Circle CI
+      - "rfc-18/**"
+    paths:
+      - "solidity/dashboard/**"
+      - ".github/workflows/token-dashboard.yml"
+
+jobs:
+  build-publish-token-dashboard-dapp:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./solidity/dashboard
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12.x"
+
+      - name: Cache resolved contracts
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-resolved-contracts
```

Comment:
```suggestion
      - name: Cache Global NPM Cache
        uses: actions/cache@v2
        env:
          cache-name: cache-global-npm-cache
```

Reply:
Thanks, I wasn't sure about that naming ;)
Fixed, see 518caea53737b1616fa46ebca92babc2db52e211.